analyses/celltype-glioblastoma/environment.yml

```
@@ -0,0 +1,16 @@
+name: "openscpca-celltype-glioblastoma"
+channels:
+  - conda-forge
+  - bioconda
+platforms:
+  - linux-64
+  - osx-64
+  - osx-arm64
+dependencies:
+  - python=3.11
+  - awscli>=2.15
+  - conda-lock>=2.5
+  - jupyter=1.0
+  - jq>=1.7
+  - pre-commit>=3.6
+  - session-info>=1.0
```

We may as well add `celltypist` (which includes `scanpy` as a dependency) to the environment at this stage. 
```suggestion
  - session-info>=1.0
  - celltypist
```